src/decentralised-development-process.md

```diff
@@ -1 +1,29 @@
+
+Both new projects and ports can receive compensation in the form of XOR, as well as development support from the SORA community. Because effectively, time and development efforts are not free, and in order to uphold the community standards, not everyone who applies will be accepted by the programme.
+
+It is also important to mention that depending on the value added by the project, it may also qualify for [Social Insurance](https://medium.com/sora-xor/social-insurance-for-systematically-important-infrastructure-18a63ef711ca), however, there will be required due diligence and the project will have to prove systematic importance to the community.
+
+It is easy to ask *wot* and *wen* when it comes to development, but *putting your RFP where your complaints are* is not that simple to do. The community has no limitation to what is proposed, however, *[spam](https://youtu.be/duFierM1yDg)*, *CEX requests*, and other off-topic requests will not be considered.
+
+**To join the SORA Builders Programme, please [fill this form](https://forms.gle/1cdvKTHLXnBXbMTr9), then join the [SORA Builders Telegram group](https://t.me/sorabuilders)**.
+
+### Time to #Buidl
+**The fate of the SORA network lies in the hands of its users!** SORA governance does not limit the use of the network to the desires of one entity, but rather, it empowers users and community members to improve the network as much as they want through the implementation of programs like **RFP submissions** and the **SORA Builders Programme**. Progress from both these initiatives will be reported in the fortnightly **Ecosystem Updates**, and all the submissions, as well as RFP and merged implementations, are publicly accessible in the [SORA RFP Github Repository](https://github.com/sora-xor/rfps) which ensures that full transparency and decentralized practices are always upheld.
+
+*For more information, read the articles SORA Decentralized Development Process, How to RFP, and [Social Insurance for Systematically Important Infrastructure](https://medium.com/sora-xor/social-insurance-for-systematically-important-infrastructure-18a63ef711ca). To stay in the loop with the latest implementations of the SORA network and Polkaswap, make sure to follow the Ecosystem updates, and finally, to get a taste of what is to come, watch the [SORA Development Public Request for Proposal (RFP) Process Github Repo!](https://github.com/sora-xor/rfps)*
+
+
+TODO move all the contents from 
```

Comment:
Sure... Please add the correct link that you wanted integrated so that I can go ahead and make the changes in the doc

Reply:
Ultimately, what I want to do is have content on the pages, and then interlinking and formatting can be done as additional tasks. This is why the PR has the bare bones of the pages. We need to discuss the headers and the proposed hierarchy too so that subsequent PR have less adjusting work to do